### PR TITLE
dai fix

### PR DESCRIPTION
**Pull request by flashfish0x**
fix error in flashloanlib  and set permissions in factory

### CODE REVIEW

**Comment by jmonteer** [comment]
Lgtm

File: contracts/LevCompFactory.sol [hunk 1]
```diff
@@ -46,6 +46,7 @@ contract LevCompFactory {
         }
 
         Strategy(newStrategy).initialize(_vault, _cToken);
+        Strategy(newStrategy).setStrategist(msg.sender);
```
Comment (fp-crypto) [inline contracts/LevCompFactory.sol:50]: The same should be done on the original created in the constructor

**Comment by fp-crypto** [comment]
Other than the one comment LGTM

**Comment by fp-crypto** [comment]
LGTM

**Review by storming0x — APPROVED** [review]
LGTM!

**Pull request merged**